models/management.cattle.io.cluster.js

```diff
@@ -168,6 +168,10 @@ export default {
     return require(`~/assets/images/vendor/${ this.providerOs }.svg`);
   },
 
+  isLocal() {
+    return this.id === 'local';
+  },
+
```

Comment:
You want `this.spec?.internal === true` here.  I don't remember exactly the situation but that's what internal means and it's possible for the id to not be `local`.  (You can then update other references to `internal` that look at `this.isLocal` instead to make them more readable)